modules/web-console/frontend/app/services/ErrorParser.service.ts

```diff
@@ -72,7 +72,8 @@ export default class {
                     const msgEnd = err.message.indexOf('[', msgStart);
                     const cause = err.message.substring(msgStart, msgEnd >= 0 && msgEnd < causeEndLine ? msgEnd : causeEndLine);
 
-                    causes.unshift(cause);
+                    if (causes && causes[0] !== cause)
```

Comment:
Lets cover ErrorParser.service.ts  with tests.

Reply:
Second this, at least add a test for the fix.

Reply:
Test has been added.